etc/testing/circle/workloads/go-test-results/grafana.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
Can you use an emptyDir in the pod volume mounts?  I don't know how much retention you're expecting to have, but maybe a read-only FS + mounting in grafana.yaml is what you're interested in for your test setup?  (I think that's what this manifest is for anyway.)